pkg/distributor/distributor_test.go

```diff
@@ -650,10 +650,26 @@ func BenchmarkShardStream(b *testing.B) {
 	allEntries := generateEntries(25000)
 
 	desiredRate := 3000
+
```

Comment:
Changes in this file aren't related to the ratestore improvements. Instead, it looks like we hadn't run benchmarks for other parts of the ratestore in awhile and they failed to run